prisma/migrations/20230210103003_usermodel/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `buyerId` to the `Ticket` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "Ticket" ADD COLUMN     "buyerId" TEXT NOT NULL;
```

Comment:
Why would the buyerId for a ticket be "TEXT NOT NULL"? Tickets do not have buyers when created.